<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fuzzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="threadCountLabel">
           <property name="text">
            <string>Ilość wątków:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="threadCountSpin">
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="crashCountLabel">
           <property name="text">
            <string>Licznik wyłożeń testowanej aplikacji:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="crashCountEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="testCountLabel">
           <property name="text">
            <string>Przeprowadzono testów:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="testCountEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="log">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="cpu1Label">
         <property name="text">
          <string>CPU1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="cpu1ProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="cpu2Label">
         <property name="text">
          <string>CPU2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="cpu2ProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="cpu3Label">
         <property name="text">
          <string>CPU3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="cpu3ProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="cpu4Label">
         <property name="text">
          <string>CPU4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="cpu4ProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
